docs/hub-deployment-guide/runbooks/phase3/initial-hub-setup.md

```diff
@@ -0,0 +1,176 @@
+(hub-deployment-guide:runbooks:phase3.1)=
+# Phase 3.1: Initial setup
+
+This phase is about configuring an initial, basic hub for the community to try it on. This hub will have basic customizations, like appearance, daskhub/basehub, a simple authentication mechanism and a list of initial admin users able to access it.
+
+## Definition of ready
+
+The following lists the information that needs to be available to the engineer before this phase can start.
+
+- Name of the hub
+- Dask gateway?
+- Splash image
+- URL of the community's webpage
+- Funded by information (name and URL)
+- Authentication Mechanism
+- List of admin users
+
+## Outputs
+
+At the end of Phase 3.1 both 2i2c engineers and the other admin users can login into the hub.
+
+The file assets that should have been generated and included in the PR should be:
+
+```bash
+➕ config/clusters/<new-cluster-name>
+  ├── common.values.yaml
+  ├── <new-hub-name>.values.yaml
+  └── enc-<new-hub-name>.secret.values.yaml
+```
+
+And the following existing file should be updated to accommodate the new hub if the cluster was recently added:
+
+```bash
+～ .github/workflows
+  └── deploy-hubs.yaml
+```
+
+```{tip}
+When reviewing initial hub setup PRs, make sure the files above are all present.
+```
+
+## Initial setup runbook
+
+All of the following steps must be followed in order to consider phase 3.1 complete. Steps might contain references to other smaller, topic-specifc runbooks that are gathered together and listed in the order they should be carried on by an engineer.
+
+1. **Determine the hub helm chart that is needed**
+
+   Use the info provided in the new hub GitHub issue for the `Dask gateway` section.
+   If Dask gateway will be needed, then go for a `daskhub` helm chart, otherwise choose a `basehub`.
+
+   Store the helm type under $HELM_CHART_TYPE env var:
+
+   ```bash
+   export HELM_CHART_TYPE=type
+   ```
+
+   ```{seealso}
+   For more information about our hub helm charts and how to choose, see [](hub-helm-charts).
+
+   See [](/topic/infrastructure/config.md) for general information about hub helm chart configuration.
+   ```
+
+2. **Determine the address of the storage server that a hub on this cluster should use to connect to it**
```

Comment:
I would swap steps (2) and (3) around so that the next, immediate thing I do once I have the server IP in my clipboard, is paste it into the deployer command to generate the config. In the current order, I have to setup files in between pulling info and running the command that requires the info.